README.md

```diff
@@ -61,11 +72,11 @@ conda deactivate
 To build the package from source, run:
 
 ```
-pip install -e .
+pip install cython  # this must be done first
```

Comment:
Would be nice to move solely to pyproject.toml and avoid this step.

Reply:
Compiling extensions is unfortunately not very pyproject.toml friendly yet.

That being said, are we sure this is necessary? We pick up build dependencies from the pyproject.toml fine in the core repository.

Reply:
There has been a bit of confusion with this, but you seem to be right, I can install in fresh environments with and without the `-e` flag. I think the problem comes down to the working directory during import, but I'm not entirely sure. Either way, will remove these "preinstall cython" statements.